spiketools/utils/check_spikes_in_ms.py

```diff
@@ -0,0 +1,24 @@
+"""Spike related checking functions."""
+
+import numpy as np
+import warnings
+
+###################################################################################################
+###################################################################################################
+
+def check_spikes_in_ms(spikes):
+    """Check if input spikes are in ms.
+
+    Parameters
+    ----------
+    spikes : 1d array
+        Spike times.
+    """
+	
+    # If there are any two spikes within the same time unit, show warning.
+    if len(np.unique((test_spikes).astype(int)))<len(np.unique(test_spikes)):
+        warnings.warn('There are 2 or more spikes within a same unit of time. Spikes might be in seconds, should be in milliseconds.')
+    # If the mean time between spikes is too low, show warning.
+    if len(spikes)>1:
```

Comment:
Spacing: add a space around operators, so it's a little easier to read. 
For example `len(spikes)>1` -> `len(spikes) > 1`. 

Can you update this on this line, and also line 19 & 23